syntax = "proto3";

package lightrpc;

// request
message RequestDescrip {
  uint32 rpc_id = 1;
  string service_name = 2;
  string method_name = 3;
}

// control message
message AuthorityMessage {
  uint32 rpc_id = 1;
  uint32 remote_key = 2;
  uint64 remote_addr = 3;
};